.github/workflows/ci.yml

```diff
@@ -115,13 +130,6 @@ jobs:
           title: "Update cassette-diff-PR${{ github.event.pull_request.number }}-${{ env.TIMESTAMP_COMMIT }}"
           body: "This PR updates the cassettes. Please merge it."
 
-
-      - name: Check PR
```

Comment:
I’m confused as to why this is getting taken out here. 

This is to create the pr with new cassettes? Or is it something else? I understand it should be removed

Reply:
I just don’t understand what it originally did and what it’s original intention was

Reply:
just display whether the cassette was created, but then I realize it's not that useful.